cmd/rm-main.go

```diff
@@ -321,6 +321,7 @@ func removeRecursive(url string, isIncomplete bool, isFake bool, olderThan int,
 
 // main for rm command.
 func mainRm(ctx *cli.Context) error {
+	fmt.Println(" ashish")
```

Comment:
We can remove the debug log prints

Reply:
Sorry ,Its a mistake.  . Removed the debug log